### ISSUE TITLE
Feature Requests Enter key

### ISSUE THREAD

**Issue by stefandanzl** [OvidijusParsiunas/active-table#18]
Hey, great work on this plugin!

Here are some ideas that would improve usability for data entry

## Enter key
I would love to be able to disable making newline characters on Enter key input.
In the meantime I solved it by wrapping the active table element in this div:
```html
<div on:keydown={handleInput}>
</div>
```
with the following javascript
```js
function handleInput(event) {
    if (event.keyCode == 13){
    event.preventDefault(); }}
```

Additionally what would be great user experience are following customizable settings on what to do on Enter key:
- Confirm your changes after input with Enter or revert back to the original cell value otherwise
- Being able to hop into the next row in the same column on Enter key, analogous to what Tab does in a row.
As far as I've seen there is no function to set the cursor to a specific cell, so I don't know how difficult that would be.

See [example](https://codesandbox.io/s/revo-grid-vanilla-tree-jsx-vyqs3?file=/src/index.js) for the result of all 3 features.

**Comment by OvidijusParsiunas** [comment]
Hey @stefandanzl!
Thankyou very much for your great suggestions!

Currently I am doing my best to fix all the minor bugs within the table + add a couple of other features I have been working on (as well as maintaining other projects), hence I will only be able to pick these up at the moment.

Nonetheless being able to focus a cell is something I have in the roadmap (Pick mode) and controlling the behaviour of the 'Enter' key is something I have considered for a while so I am very eager to implement this when I can. In regards to confirming input - it would be quite a big change, hence I will only consider this once I have all other features implemented.

Thankyou very much for the great suggestions once again, I will be closing this story for now and will re-open it once I start working on the features listed here.

If you experience any issues/bugs or have other suggestions, feel free to create a new issue. Thanks!

**Comment by OvidijusParsiunas** [comment]
Hi, the Enter key behaviour to move down to row below has is now available in version [1.1.2](https://github.com/OvidijusParsiunas/active-table/releases/tag/1.1.2).
